application/config/config.php

```diff
@@ -23,7 +23,7 @@
 | a PHP script and you can easily do that on your own.
 |
 */
-$config['base_url'] = '';
```

Comment:
Can you please revert this line of change. The base_url should be automatically detected. This may cause some issues under the developer's local development environment but let's try to stick with an empty string on the master branch.